### DISCUSSION TITLE
Server maintenance on Tue 2023-03-16

### DISCUSSION THREAD

**Discussion by dmtri35** [WATonomous/infrastructure-support#24] [General Announcements]
Hey @WATonomous/compute-cluster-users,

We will be having our weekly maintenance on Tue 2023-03-16 from 4pm to 6pm. The main goals are to replace a faulty RTX 3090 in `trpro` with a brand new RTX 4090 and to upgrade the connection to 10Gbps. The cluster might experience some down time so we strongly suggest you to not schedule any training during this time period.

Thank you for your patience,
Tri

**Comment by ben-z** [comment]
Multi-gigabit internet is beautiful 😍 

<img width="703" alt="image" src="https://github.com/WATonomous/infrastructure-support/assets/5977478/3ec25cae-fb54-4359-b4b6-a1d22516e344">

**Comment by dmtri35** [comment]
![image](https://github.com/WATonomous/infrastructure-support/assets/87549865/12e9358d-6ff6-4068-ad44-7a6487ce1f30)
We now have 2x4090 + 4x3090 on `trpro-ubuntu1`

**Reply by ben-z** [reply]
Doing some stress test on these to make sure our power capacity is sufficient. Should be done in 9 hours:
<img width="644" alt="image" src="https://github.com/WATonomous/infrastructure-support/assets/5977478/4ae2d493-299c-4885-982f-42a8f0ca4fad">

**Comment by dmtri35** [comment]
It's a bit longer than what I've expected but everything is green now on the [status page](https://status.watonomous.ca/grouped) 🎉
